src/Integration/Transition/MuteWindowDialog.xaml.cs

```diff
@@ -44,6 +46,7 @@ public MuteWindowDialog(bool showAccept)
             SetVisibility(showAccept);
 
             Transitions = InitializeTransitions();
+            Borders = InitializeBorders();
```

Comment:
IDK if borders is a good name. It doesn't tell what this is used for

Reply:
Maybe BorderRadioButtons?